src/components/PreviewComment/index.js

```diff
@@ -60,7 +60,7 @@ const PreviewComment = ({comment, time, image, totalComment, onPress, user, item
                     style={{
                       width: 24,
                       height: 24,
-                      backgroundColor: COLORS.white,
+                      backgroundColor: 'white',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the hardcoded color 'white' should be replaced with a constant.
```diff
- backgroundColor: 'white',
+ backgroundColor: colors.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->